File: Csharp/URI/01-Iniciantes/02-Nivel/1009.cs
using System;    

namespace Uri1009.URI
{
    public class Uri1009
    {
        static void Main()
        {
            
            string nameEmployer;
            double totalComission, salary , totalSales;
            salary = totalSales = 0;            

            nameEmployer = ReceiveNameEmployer();
            ReceiveSalaryData( ref salary , ref totalSales);
            totalComission = CalculateCommission(totalSales);
            Console.WriteLine("TOTAL = R$ {0:0.00}", (salary + totalComission));
        }
        static string ReceiveNameEmployer()
        {
        string nameEmployer;    
        Console.WriteLine("Digite o nome do funcionario:");
        nameEmployer = Console.ReadLine();    
        return nameEmployer;
        }

        static void ReceiveSalaryData(ref double salary, ref double totalSales)
        {
        Console.WriteLine("Digite o salario atual");
        salary = double.Parse(Console.ReadLine());
        Console.WriteLine("Digite as vendas nesse mes:");
        totalSales = double.Parse(Console.ReadLine());        
        
        
        }
        static double CalculateCommission(double totalSales)
        {
        double commissionTax , totalCommission;
        commissionTax = 0.15;
        totalCommission = ( totalSales * commissionTax);    
        return totalCommission;
        }
    }  
        
}
    

File: Cplusplus/URI/01-iniciantes/02-Nivel/1009.cpp
#include <iostream>
#include <iomanip>
#include <string>

using namespace std;

string ReceiveNameEmployer();
double ReceiveSalaryData(double salaryData[2]);
double CalculateCommission(double salaryData[2]);

int main()
{
    string nameEmployer;
    double salaryData[2] , totalComission;
    
    nameEmployer = ReceiveNameEmployer();
    ReceiveSalaryData(salaryData);
    totalComission = CalculateCommission(salaryData);
    cout << "TOTAL = R$ " << fixed << setprecision(2)  << (salaryData[0] + totalComission) << endl;
    
    
    return 0;

}
string ReceiveNameEmployer()
{
    string nameEmployer;    
    cout << "Digite o nome do funcionario" << endl;
    cin >> nameEmployer;    
    return nameEmployer;
}

double ReceiveSalaryData(double salaryData[2])
{
    double salary, totalSales;
    
    cout << "Digite o salario do funcionario" << endl;
    cin >> salary;  
    cout << "Digite o salario do funcionario" << endl;
    cin >> totalSales;
    salaryData[0] = salary;
    salaryData[1] = totalSales; 
    return salaryData[2];
}
double CalculateCommission(double salaryData[2])
{
    double commissionTax , totalCommission;
    commissionTax = 0.15;
    totalCommission = ( salaryData[1] * commissionTax);    
    return totalCommission;
}
